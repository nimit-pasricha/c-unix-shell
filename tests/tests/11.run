../src/wsh tests/11.wsh