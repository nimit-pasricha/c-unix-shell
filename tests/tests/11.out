/tmp
/tmp/testcd
/tmp/testcd/subdir
/tmp/testcd/subdir
/home/nimit
/tmp
